IOS Run problem: OIDExternalUserAgentIOSNoSSO.h:27:52: warning: cannot find protocol definition for 'OIDExternalUserAgent'

```
flutter_appauth.framewor
    k/Headers/OIDExternalUserAgentIOSNoSSO.h:27:52: warning: cannot find protocol definition for 'OIDExternalUserAgent'
    @interface OIDExternalUserAgentIOSNoSSO : NSObject<OIDExternalUserAgent>
                                                       ^
    flutter_appauth.framewor
    k/Headers/OIDExternalUserAgentIOSNoSSO.h:22:11: note: protocol 'OIDExternalUserAgent' has no definition
    @protocol OIDExternalUserAgent;
              ^
    1 warning generated.
    1 warning generated.

```

--- a/flutter_appauth/CHANGELOG.md
+++ b/flutter_appauth/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 0.9.1+1
+
+* Fixes warnings picked up in Xcode raised in issue [123](https://github.com/MaikuB/flutter_appauth/issues/123)
+
 ## 0.9.1
 
 * [iOS] Update AppAuth SDK dependency to 1.3.1

--- a/flutter_appauth/ios/Classes/FlutterAppauthPlugin.m
+++ b/flutter_appauth/ios/Classes/FlutterAppauthPlugin.m
@@ -197,7 +197,7 @@ NSString *const AUTHORIZE_ERROR_MESSAGE_FORMAT = @"Failed to authorize: %@";
     }
 }
 
-- (NSObject<OIDExternalUserAgent> *)userAgentWithViewController:(UIViewController *)rootViewController useEphemeralSession:(BOOL)useEphemeralSession {
+- (id<OIDExternalUserAgent>)userAgentWithViewController:(UIViewController *)rootViewController useEphemeralSession:(BOOL)useEphemeralSession {
     if (useEphemeralSession) {
         return [[OIDExternalUserAgentIOSNoSSO alloc]
                 initWithPresentingViewController:rootViewController];

--- a/flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.h
+++ b/flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.h
@@ -17,9 +17,9 @@
  */
 
 #import <UIKit/UIKit.h>
+#import "OIDExternalUserAgent.h"
 
 @class SFSafariViewController;
-@protocol OIDExternalUserAgent;
 
 NS_ASSUME_NONNULL_BEGIN
 

--- a/flutter_appauth/pubspec.yaml
+++ b/flutter_appauth/pubspec.yaml
@@ -1,6 +1,6 @@
 name: flutter_appauth
 description: This plugin provides an abstraction around the Android and iOS AppAuth SDKs so it can be used to communicate with OAuth 2.0 and OpenID Connect providers
-version: 0.9.1
+version: 0.9.1+1
 homepage: https://github.com/MaikuB/flutter_appauth/tree/master/flutter_appauth
 
 environment:

